modules/javafx.graphics/src/main/resources/com/sun/prism/es2/glsl/main.vert

```diff
@@ -37,12 +37,17 @@ struct Light {
     vec3 color;
     vec3 attn;
     float range;
+    vec3 dir;
```

Comment:
These need to be reordered to match the changes in the fragment shaders. As it is, it causes a shader link error.

Reply:
Sorry, I can't test the fix on Linux for now. I hope made the change, but didn't verify that it works.